addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -15,11 +15,13 @@
 
 params ["_logic"];
 
+if !(local _logic) exitWith {};
```

Comment:
Yep, this line is standard in all zeus modules (BI included) with a few exceptions.